bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJetsToQQ_5f_NLO_FXFX/ZJetsToQQ_5f_NLO_FXFX_run_card.dat

```diff
@@ -0,0 +1,146 @@
+#***********************************************************************
+#                        MadGraph5_aMC@NLO                             *
+#                                                                      *
+#                      run_card.dat aMC@NLO                            *
+#                                                                      *
+#  This file is used to set the parameters of the run.                 *
+#                                                                      *
+#  Some notation/conventions:                                          *
+#                                                                      *
+#   Lines starting with a hash (#) are info or comments                *
+#                                                                      *
+#   mind the format:   value    = variable     ! comment               *
+#***********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#
+#***********************************************************************
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of events (and their normalization) and the required          *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+  1500 = nevents ! Number of unweighted events requested 
+ 0.001 = req_acc ! Required accuracy (-1=auto determined from nevents)
+    20 = nevt_job! Max number of events per job in event generation. 
+                 !  (-1= no split).
+average = event_norm ! Normalize events to sum or average to the X sect.
```

Comment:
change the average to bias

Reply:
That's already done here https://github.com/cmantill/genproductions/tree/mg26x/bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJetsToQQ_5f_NLO_FXFX_bias 

I uploaded both sets of cards (with and without bias weight) - I'm not sure this has been the inclusive one has been produced before (or if so I could not find the old cards for NLO).

Reply:
I think this change is still needed, cfr https://github.com/cms-sw/genproductions/pull/1621/files#diff-2c6138a3a9aaa18acaa48176ae3137dbR32